requirements/app/base.txt

```diff
@@ -1,7 +1,7 @@
 lightning-cloud==0.5.3
 packaging
 deepdiff>=5.7.0, <=5.8.1
-starsessions
+starsessions==1.3
```

Comment:
Can we give some range?

Reply:
```suggestion
starsessions>=1.0, <2.0
```

Reply:
we should revert it soon and upgrade it to 2.0. I expect that to happen when we work on multitenancy.